web-react-ts/src/global-utils.ts

```diff
@@ -195,44 +195,68 @@ export const debounce = (callback: () => void, delay = 500) => {
 }
 
 interface MemberWithTitle extends Member {
-  titleConnection: {
-    edges: {
-      node: { title: 'Pastor' | 'Reverend' | 'Bishop' }
-    }[]
-  }
+  // titleConnection: {
+  //   edges: {
+  //     node: { title: 'Pastor' | 'Reverend' | 'Bishop' }
+  //   }[]
+  // }
```

Comment:
Is this supposed to be here ??

Reply:
Well. We may want to use it later. So I'm uncommenting it